tests/src/test/scala/whisk/core/cli/test/WskEntitlementTests.scala

```diff
@@ -214,11 +214,11 @@ abstract class WskEntitlementTests extends TestHelpers with WskTestHelpers with
 
       val stdout = wsk.pkg.get(name)(defaultWskProps).stdout
       val annotationString = wsk.parseJsonString(stdout).fields("annotations").toString
-      annotationString should include regex (""""key":"a"""")
-      annotationString should include regex (""""value":"A"""")
-      annotationString should include regex (s""""key":"${WhiskPackage.bindingFieldName}"""")
-      annotationString should not include regex(""""key":"xxx"""")
-      annotationString should include regex (s""""name":"${samplePackage}"""")
+      annotationString should include(""""key":"a"""")
+      annotationString should include(""""value":"A"""")
+      annotationString should include(s""""key":"${WhiskPackage.bindingFieldName}"""")
+      annotationString should not include (""""key":"xxx"""")
+      annotationString should include(s""""name":"${samplePackage}"""")
```

Comment:
If we're touching this anyway: Can we assert on the json itself and not `toString` the whole thing?